File: on_demand_flight_status/v2/get/Node SDK/on_demand_flight_status.js
const Amadeus = require("amadeus");

const amadeus = new Amadeus({
  clientId: "YOUR_API_KEY",
  clientSecret: "YOUR_API_SECRET",
});

async function main() {
  try {
    // What's the current status of my flight?
    const response = await amadeus.schedule.flights.get({
      carrierCode: "AZ",
      flightNumber: "319",
      scheduledDepartureDate: "2022-03-13",
    });

    console.log(response);
  } catch (error) {
    console.error(error);
  }
}

main();

File: on_demand_flight_status/v2/get/Python SDK/on_demand_flight_status.py
from amadeus import Client, ResponseError

amadeus = Client(
    client_id='YOUR_AMADEUS_API_KEY',
    client_secret='YOUR_AMADEUS_API_SECRET'
)

try:
    '''
    Returns flight status of a given flight
    '''
    response = amadeus.schedule.flights.get(carrierCode='AZ',
                                            flightNumber='319',
                                            scheduledDepartureDate='2022-03-13')
    print(response.data)
except ResponseError as error:
    raise error
